test/integration/kubernetes_deploy_test.rb

```diff
@@ -154,9 +158,9 @@ def test_dynamic_erb_collection_works
     assert_equal ["web-one", "web-three", "web-two"], deployments.map { |d| d.metadata.name }.sort
   end
 
-  # Reproduces k8s bug
+  # The next three tests reproduce a k8s bug - These problems should be caught during dry run
```

Comment:
Nit: `should be caught during dry run` => `should be caught during dry run but Kubernetes doesn't catch them`